app/src/main/java/de/ph1b/audiobook/features/settings/dialogs/SortingModeDialogFragment.kt

```diff
@@ -0,0 +1,46 @@
+package de.ph1b.audiobook.features.settings.dialogs
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import com.afollestad.materialdialogs.MaterialDialog
+import de.ph1b.audiobook.R
+import de.ph1b.audiobook.features.bookOverview.list.BookComparator
+import de.ph1b.audiobook.injection.App
+import de.ph1b.audiobook.injection.PrefKeys
+import de.ph1b.audiobook.persistence.pref.Pref
+import javax.inject.Inject
+import javax.inject.Named
+
+/**
+ * Dialog for picking the UI theme.
+ */
+class SortingModeDialogFragment : DialogFragment() {
```

Comment:
This should extend `DialogController`.

Reply:
How does this work then? All the other settings dialogs extend from `DialogFragment()`.

Reply:
Like here:
https://github.com/PaulWoitaschek/Voice/blob/0bd5aa9bb2b0d71e59a68a71aa17b855bc52c2ab/app/src/main/java/de/ph1b/audiobook/features/bookmarks/dialogs/DeleteBookmarkDialog.kt

I'll open a separate ticket for migrating the others.